api/v1alpha1/environmentrequest_types.go

```diff
@@ -61,8 +82,10 @@ type EnvironmentRequestSpec struct {
 	// TODO: Dataset per provider?
 	Dataset *apiextensionsv1.JSON `json:"dataset,omitempty"`
 
-	Providers EnvironmentProviders `json:"providers"`
-	Splitter  Splitter             `json:"splitter"`
+	Providers          EnvironmentProviders        `json:"providers"`
+	Splitter           Splitter                    `json:"splitter"`
+	ServiceAccountName string                      `json:"serviceaccountname,omitempty"`
+	JobConfig          EnvironmentRequestJobConfig `json:"jobConfig,omitempty"`
```

Comment:
I would rather have this be called `Config`. We might not want to use jobs in the underlying structure in the future and `jobConfig` might confuse users into thinking they can add job specific parameters.